addons/account/static/src/components/x2one_button/x2one_button.js

```diff
@@ -0,0 +1,54 @@
+import { _t } from "@web/core/l10n/translation";
```

Comment:
This seems like just a copy of the x2many_button.js?  Not sure what the goal is then. (I even wonder if we can not do everything with the actionable errors widget (ActionableErrors), but maybe that leads us too far off...)